charts/studio/values.yaml

```diff
@@ -29,15 +29,15 @@ global:
   # -- Studio: Additional environment variables for all pods
   envVars: {}
   # Example:
-  # envVars: 
+  # envVars:
   #   DEBUG: "True"
 
   blobvault:
     # -- Blobvault local backing store size
     persistentVolume:
-      accessModes: 
+      accessModes:
         - ReadWriteOnce
-      storageClassName: local-path
+      # storageClassName: local-path
```

Comment:
This will break AMI builds and such. If we want to avoid this, we need to design some system for merging the configs in layers (e.g., vendor (Helm chart) + system (AMI) + user)

Reply:
@jesper7  You mean, on AMI there is no default StorageClass?

Reply:
I checked the AMI, storage clas is a default one and ☝️ change is not breaking for it:
```bash
ubuntu@ip-10-20-59-216:~$ kubectl get pod,pvc,pv
NAME              READY   STATUS    RESTARTS   AGE
pod/task-pv-pod   1/1     Running   0          35s

NAME                                  STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
persistentvolumeclaim/task-pv-claim   Bound    pvc-23c38883-97a0-49fb-956d-4c71b3ca6fed   3Gi        RWO            local-path     64s

NAME                                                        CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                   STORAGECLASS   REASON   AGE
persistentvolume/pvc-23c38883-97a0-49fb-956d-4c71b3ca6fed   3Gi        RWO            Delete           Bound    default/task-pv-claim   local-path              31s
```
I created `PVC` and `Pod` from https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/, `PV` was created automatically

---
StorageClass yaml
```yaml
# Please edit the object below. Lines beginning with a '#' will be ignored,
# and an empty file will abort the edit. If an error occurs while saving this file will be
# reopened with the relevant failures.
#
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  annotations:
    objectset.rio.cattle.io/applied: H4sIAAAAAAAA/4yRT+vUMBCGv4rMua1bu1tKwIOu7EUEQdDzNJlux6aZkkwry7LfXbIqrIffn2PyZN7hfXIFXPg7xcQSwEBSiXimaupSxfJ2q6GAiYMDA9/+oKPHlKCAmRQdKoK5AoYgisoSUj5K/5OsJtIqslQWVT3lNM4xUDzJ5VegWJ63CQxMTXogW128+czBvf/gnIQXIwLOBAa8WPTl30
    objectset.rio.cattle.io/id: ""
    objectset.rio.cattle.io/owner-gvk: k3s.cattle.io/v1, Kind=Addon
    objectset.rio.cattle.io/owner-name: local-storage
    objectset.rio.cattle.io/owner-namespace: kube-system
    storageclass.kubernetes.io/is-default-class: "true" # HERE <<<<<<< 
  creationTimestamp: "2023-05-31T12:59:31Z"
  labels:
    objectset.rio.cattle.io/hash: 183f35c65ffbc3064603f43f1580d8c68a2dabd4
  name: local-path
  resourceVersion: "272"
  uid: eec8afdf-a6a6-4edb-b1e4-dbe90c85c8b0
provisioner: rancher.io/local-path
reclaimPolicy: Delete
volumeBindingMode: WaitForFirstConsumer
~
```

Description:
![image](https://github.com/iterative/helm-charts/assets/5058132/c6d116fb-32a4-4025-9565-e594c401b4b1)